.travis.yml

```diff
@@ -31,9 +31,11 @@ install:
 - go get github.com/FiloSottile/gvt
 - go get -u github.com/alecthomas/gometalinter
 - gometalinter --install
+- curl -o ~/bin/shfmt -sSL https://github.com/mvdan/sh/releases/download/v2.0.0/shfmt_v2.0.0_linux_amd64
+- chmod +x ~/bin/shfmt
+- shfmt -version
 - git clone --branch v1.3.1 https://github.com/mvdan/sh.git "${GOPATH%%:*}/src/github.com/mvdan/sh"
 - go get github.com/mvdan/sh/...
```

Comment:
These two lines that install `shfmt` again from the `v1.3.1` branch will have to be removed 👍

Reply:
Verdammt!! TY